app/views/provider/admin/messages/inbox/show.html.erb

```diff
@@ -1,5 +1,3 @@
-<% content_for :page_header_title, 'Message' %>
```

Comment:
We already have the same exact line inside the `provider/admin/messages/message` template.

Reply:
So should it be removed from the template and ensure that it is always there by having it here? Just asking

Reply:
The `provider/admin/messages/message` partial is used in 2 other templates, so I guess it's better having it three to avoid duplication.

Reply:
I meant the opposite - remove from templates and keep in the partial (here). But either way, do as you feel

Reply:
I'm not sure we are understsanding each other :grimacing: 

So there is a partial with the header included https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/views/provider/admin/messages/_message.html.erb#L1 and it is used in:
https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/views/provider/admin/messages/inbox/show.html.erb#L3
https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/views/provider/admin/messages/trash/show.html.erb#L1
https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/views/provider/admin/messages/outbox/show.html.erb#L1

In addition, [porta/app/views/provider/admin/messages/inbox/show.html.erb](https://github.com/3scale/porta/blob/bcbaea6392b58a89291932b2289127e44864e288/app/views/provider/admin/messages/inbox/show.html.erb#L3) had its own `<% content_for :page_header_title, 'Message' %>` which was causing a duplication, and I removed that one.